I.write-tests/01-greet-people/greet-people.js

```diff
@@ -1,18 +1,18 @@
-function greetPeople(people) {
-    var greeting = 'Hello ';
+function greetPeople (people) {
+  var greeting = 'Hello '
+  return greeting + people.join(', ');
```

Comment:
Well done for using `join` :clap: 
Do we need the comments below? If they are not adding value to the code, we should remove it :)

Reply:
noted thanks